src/openforms/authentication/api/serializers.py

```diff
@@ -29,6 +29,11 @@ class LoginLogoSerializer(serializers.Serializer):
         help_text=_("Information link to the authentication provider"),
         read_only=True,
     )
+    appearance = serializers.URLField(
```

Comment:
why is this a `URLField`? it's a field that only has to possible choices: `dark` or `light`, and it should be output as `enum` in the API schema documentation.